pkg/git/git.go

```diff
@@ -1298,3 +1300,14 @@ func (e NetworkError) CanRetry() bool {
 	// Otherwise permanent
 	return false
 }
+
+func safeError(err error) error {
+	if err == nil {
+		return nil
+	}
+	// To parse the GitHub ApiKey
+	re := regexp.MustCompile(`[0-9a-zA-Z]{35,40}`)
```

Comment:
Make this please a global var to avoid re-compiling every time.

Reply:
ok

thanks for your review